src/collective/nitf/profiles/default/registry.xml

```diff
@@ -77,4 +77,10 @@
       <value key="group" i18n:translate="">NITF</value>
   </records>
 
+  <record name="plone.app.tiles">
```

Comment:
no, but... collective.nitf is installed :-)

what we need is to remove that at uninstall time.